packages/now-cli/scripts/preinstall.js

```diff
@@ -57,6 +74,11 @@ async function main() {
     );
```

Comment:
Let's also update `Now CLI` on line 56 to be `Vercel CLI` and any other references of Now in this file.